.github/workflows/container-build-push.yaml

```
@@ -52,12 +42,14 @@ jobs:
             echo "tag=${{ steps.branch-name.outputs.branch }}" >> $GITHUB_OUTPUT
           fi
 
-      - name: Build and push the container image
-        id: build_and_push_model_image
-        uses: docker/build-push-action@v6
+      - name: Docker Login
+        uses: docker/login-action@v3
         with:
-          no-cache: true
-          push: true
-          tags: |
-            ${{ vars.CONTAINER_REGISTRY_URL}}/${{ env.IMAGE_NAME }}:${{ steps.image-tag.outputs.tag }}
-          file: Containerfile
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Docker build and push
+        run: |
+          chmod +x $GITHUB_WORKSPACE/.github/scripts/docker_build_and_push.sh
+          $GITHUB_WORKSPACE/.github/scripts/docker_build_and_push.sh ${{ env.IMAGE_NAME }} ${{ steps.image-tag.outputs.tag }}
```

If you want to avoid having to make the script executable explicitly, why not just?
```suggestion
         bash $GITHUB_WORKSPACE/.github/scripts/docker_build_and_push.sh ${{ env.IMAGE_NAME }} ${{ steps.image-tag.outputs.tag }}
```